server/services/prismic-content.js

```diff
@@ -0,0 +1,180 @@
+import type {Article} from '../model/article';
+import type {Picture} from '../model/picture';
+import Prismic from 'prismic-javascript';
+import {RichText, Date as PrismicDate} from 'prismic-dom';
+import {prismicApiV2} from './prismic-api';
+
+export async function getContent(id) {
+  const prismic = await prismicApiV2();
+  const fetchLinks = [
+    'people.name', 'people.image', 'people.twitterHandle',
+    'books.title', 'books.title', 'books.author', 'books.isbn', 'books.publisher', 'books.link', 'books.cover',
+    'series.name','series.description', 'series.color', 'series.commissionedLength'
+  ];
+  const articles = await prismic.query(Prismic.Predicates.at('document.id', id), {fetchLinks});
+  const prismicArticle = articles.total_results_size === 1 ? articles.results[0] : null;
+
+  // TODO : construct this not from strings
+  const url = `/articles/${id}`;
+
+  // TODO: Leave this in the flow of the body
+  const prismicStandfirst = prismicArticle.data.body.find(slice => slice.slice_type === 'standfirst');
+  const standfirst = prismicStandfirst && RichText.asText(prismicStandfirst.primary.text);
+
+  // TODO: Add this to the article body
+  // TODO: potentially get rid of this
+  const publishDate = PrismicDate(prismicArticle.data.publishDate || prismicArticle.first_publication_date);
+
+  // TODO:
+  const mainMedia = prismicArticle.data.body.filter(slice => slice.primary.weight === 'featured').map(slice => {
+    return prismicImageToPicture(slice.primary);
+  });
+
+  // TODO: Don't convert this into thumbnail
+  const promo = prismicArticle.data.promo.find(slice => slice.slice_type === 'embeddedImage').primary;
+  const thumbnail = prismicImageToPicture(promo);
+
+  // TODO: Support more than 1 author
+  // TODO: Support creator's role
+  // TODO: For some reason person.image isn't coming through
+  const creator = prismicArticle.data.creators.find(creator => creator.slice_type === 'person');
+  const person = creator && creator.primary.person.data;
+  const author = person && {
+    name: person.name,
+    twitterHandle: person.twitterHandle,
+    image: person.image.url
+  };
+
+  const series = [] || prismicArticle.data.series.length > 0 && prismicArticle.data.series.map(prismicSeries => {
+      const seriesData = prismicSeries.primary.series.data;
+      return {
+        name: seriesData.name,
+        description: seriesData.description,
+        commissionedLength: seriesData.commissionedLength,
+        positionInSeries: prismicSeries.primary.positionInSeries
+      };
+    });
```

Comment:
What does this do? Doesn't `[] || anything` always return `[]` and not evaluate `anything`?